package.json

```diff
@@ -10,11 +10,6 @@
     "start": "next start",
     "lint": "next lint"
   },
-  "engines": {
```

Comment:
Engines shouldn't effect tailwind. Let's keep enforcing yarn, we'll get into weird dependency errors later if not. I will look into this